README.md

```diff
@@ -18,10 +19,18 @@ the team repos (located in `assets/repo-list`).
 
 To link the dotfiles and install system-level packages, just run:
 
-```
+```bash
 ./install.sh
 ```
 
+On OS X
+
+```bash
+./install-on-osx.sh
+```
+
+Run `:PlugInstall` in vim
```

Comment:
Btw, you can automate this as well.  This should work for `nvim`, though I haven't tried it.

```bash
vim -u PATH_TO_VIM_CONFIG +PluginInstall +qa -
vim -u PATH_TO_VIM_CONFIG +PluginClean! +qa -
```

Reply:
I see it here: https://github.com/cloudfoundry/log-cache-dotfiles/pull/5/files#diff-44d0c44c9348f52ac00a8ca2f28746e6R58.  Maybe this step in the readme isn't needed?

Reply:
Yeah, I thought it was done, but it didn't do anything or work until I ran it. Another thing I did manually was set up go with homebrew (since I had it already installed).